app/src/features/rules/routes.tsx

```diff
@@ -50,35 +50,79 @@ export const ruleRoutes: RouteObject[] = [
   {
     path: `${PATHS.RULES.INDEX}`,
     element: <RulesContainerV2 />,
+    handle: {
+      breadcrumb: {
+        label: "Rules",
+        navigateTo: PATHS.RULES.INDEX,
```

Comment:
Instead of navigateTo, can't we just use the path?

Reply:
Yes, updated